docs/README.md

```diff
@@ -18,27 +18,27 @@ The following JSON is the default configuration for the module and can be overri
 					"metadata": {
 						"opengraph": true,
 						"twitter": true,
-						"json-ld": true,
-						"fallback-image": "",
+						"fallback-image": false,
+						"fallback-image-id": "",
+						"pinterest-verify": false,
 						"social-urls": {
-							"google": "",
 							"facebook": "",
 							"twitter": "",
 							"instagram": "",
-							"youtube": "",
 							"linkedin": "",
 							"myspace": "",
 							"pinterest": "",
-							"soundcloud": "",
-							"tumblr": "",
+							"youtube": "",
+							"wikipedia": ""
 						},
+						"opengraph-fallback": {
+							"og-frontpage-title": "",
```

Comment:
I'm actually wondering if these settings are used or even needed. They feel like legacy options. In my testing, I couldn't find anything that actually used them. What's your thought about including the config options in the `load.php` file but just not documenting them? I.e. it's there for backpat but you probably wouldn't need it? Too much cruft? Should we just remove entirely if it doesn't look like something that's in use?

Reply:
I'll leave them so they can be used but drop the prefix. These look like they are mostly for sites where the homepage displays blog posts rather than a static front page.